it/src/main/java/org/corfudb/universe/UniverseManager.java

```diff
@@ -310,7 +310,7 @@ public void shutdown() {
         }
 
         private void checkMode(UniverseMode expected) {
-            if (universeMode != expected) {
+            if (!universeMode.equals(expected)) {
```

Comment:
please don't use "equals" for enums, so, please change it back

Reply:
Can you clarify it? I went through some discussion about this topic https://stackoverflow.com/questions/1750435/comparing-java-enum-members-or-equals
I'm a little inclined to use equals here though, as if one of the variable is null, we may expect a NPE instead of IllegalStateException?

Reply:
equals is too much, it's overkill. The same way you can check numbers: `Integer.of(1).equals(1)`. 
Anytime when you use algebraic data types it's better to use algebraic operators.

The same works for boolean, imagine this: `if(table.isEmpty().equals(true))` the code looks awkward